app_dart/test/request_handlers/update_existing_flaky_issues_test_data.dart

```diff
@@ -203,66 +207,82 @@ https://flutter-dashboard.appspot.com/#/build?taskFilter=Linux%20ci_yaml_flutter
 Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fixing-flaky-tests to fix the flakiness and enable the test back after validating the fix (internal dashboard to validate: go/flutter_test_flakiness).
 ''';
 
-const String ciYamlContent = '''
-# Describes the targets run in continuous integration environment.
-#
-# Flutter infra uses this file to generate a checklist of tasks to be performed
-# for every commit.
-#
-# More information at:
-#  * https://github.com/flutter/cocoon/blob/master/scheduler/README.md
-enabled_branches:
-  - master
-
-targets:
-  - name: Mac_android android_semantics_integration_test
-    builder: Mac_android android_semantics_integration_test
-    presubmit: false
-    scheduler: luci
-    properties:
-      tags: >
-        ["devicelab"]
-
-  - name: Mac_android ignore_myflakiness
-    builder: Mac_android android_semantics_integration_test
-    presubmit: false
-    scheduler: luci
-    properties:
-      ignore_flakiness: "true"
-      tags: >
-        ["devicelab"]
-
-  - name: Linux ci_yaml flutter roller
-    recipe: infra/ci_yaml
-    bringup: true # TODO(chillers): https://github.com/flutter/flutter/issues/93225
-    timeout: 30
-    properties:
-      tags: >
-        ["framework","hostonly","shard"]
-    scheduler: luci
-    runIf:
-      - .ci.yaml
-
-  - name: Mac build_tests_1_4
-    recipe: flutter/flutter_drone
-    timeout: 60
-    properties:
-      add_recipes_cq: "true"
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:29.0"},
-          {"dependency": "chrome_and_driver", "version": "version:84"},
-          {"dependency": "open_jdk", "version": "11"},
-          {"dependency": "xcode", "version": "13a233"},
-          {"dependency": "gems", "version": "v3.3.14"},
-          {"dependency": "goldctl", "version": "git_revision:3a77d0b12c697a840ca0c7705208e8622dc94603"}
-        ]
-      shard: build_tests
-      subshard: "1_4"
-      tags: >
-        ["framework","hostonly","shard"]
-    scheduler: luci
-''';
+final CiYaml testCiYaml = CiYaml(
```

Comment:
Thanks for the cleanup! This has been broken a few times because it was parsing the JSON instead of the proto, and it's great to see it won't be as brittle anymore!

Reply:
Happy to make the changes. Thank you!